stable/external-dns/values.yaml

```diff
@@ -280,6 +280,9 @@ annotationFilter: ""
 ## When enabled, prints DNS record changes rather than actually performing them
 ##
 dryRun: false
+## When enabled, triggers run loop on create/update/delete events (optional, in addition of regular interval)
+##
+events: false
```

Comment:
Could we give it a more intuitive name like `triggerLoopOnEvent` ?

Reply:
`events` is the name of the [external-dns argument](https://github.com/kubernetes-sigs/external-dns/blob/master/pkg/apis/externaldns/types.go#L387). To be intuitive  _and_ consistent, I could submit a PR to rename the external-dns argument from `--events` to `--triggerLoopOnEvent`, but there is no guarantee it'd go in before the next release of external-dns, which would be the _first_ release to support the event based functionality.  

Or are you suggesting we just have different names between helm and external-dns, which seems counterintuitive to me.

Reply:
In my opinion, the helm chart should be like an abstraction that represent different ways to deploy an application/service, I mean, if we say
```
helm install myrelease --set events=true stable/external-dns
```
then It says nothing to me, at least it says the same than the `cli` to deploy the application, I mean, that's ok if you know the application and its `cli` behaviour is familiar to you but if you don't.

On the other hand, if we say
```
helm install myrelease --set triggerLoopOnEvent=true stable/exteranl-dns
```

then we know better what kinds of deployment we are doing, I mean, the chart is given a big value over the application deployment just using three more words.

Don't you think so?

Reply:
Sounds good to me. Ill make the change.

Reply:
Done. `events` chart option renamed to `triggerLoopOnEvent`.